Assets/Scripts/StateMachines/PlushieDamageSM.cs

```diff
@@ -0,0 +1,72 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlushieDamageSM : BaseStateMachine
+{
+    // State of plushie damage as a small rip
+    [HideInInspector]
+    public SmallRipState smallRipState;
+
+    // State of plushie damage as a large rip
+    [HideInInspector]
+    public LargeRipState largeRipState;
+
+    // State of plushie damage as a worn stuffing
+    [HideInInspector]
+    public WornStuffingState wornStuffingState;
+
+    // State of plushie damage as a large rip with missing stuffing
+    [HideInInspector]
+    public MissingStuffingState missingStuffingState;
+
+    // Goal state of plushi damage - when repair on a damage is finished
+    [HideInInspector]
+    public RepairFinishState repairFinishState;
+
+    // Reference to PlushieDamageGO component
+    internal PlushieDamageGO _plushieDamageGO;
+    
+    
+    // Initialize fields on load
+    private void Awake() {
+        InitializeFields();
+    }
+
+    // Initialize references before first frame update
+    private void Start() {
+        base.InitializeState();
+        InitializeReferences();
+    }
+
+    // Initialize fields in this class
+    private void InitializeFields() {
+        InitializeStates();
```

Comment:
I would recommend renaming the InitizleStates function to SetupStateMachine or something, and just call that for now until we need to break out more layers like this. Will make it easier to see at a glance.